<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="3" >
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Files</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLineEdit" name="lineEdit_Source" />
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="pushButton_Open" >
         <property name="text" >
          <string>File 1</string>
         </property>
         <property name="icon" >
          <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLineEdit" name="lineEdit_Source2" />
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="pushButton_Open2" >
         <property name="text" >
          <string>File 2</string>
         </property>
         <property name="icon" >
          <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="pushButton_Calc" >
      <property name="text" >
       <string>get md5sum</string>
      </property>
      <property name="icon" >
       <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/roll.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Result</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLineEdit" name="lineEdit_Result" />
       </item>
       <item row="1" column="0" >
        <widget class="QLineEdit" name="lineEdit_Result2" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_Match" >
         <property name="text" >
          <string>match</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>301</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>true</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionSave" />
   <addaction name="actionInfo" />
   <addaction name="actionQuit" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>new</string>
   </property>
  </action>
  <action name="actionInfo" >
   <property name="icon" >
    <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/messagebox_info.png</iconset>
   </property>
   <property name="text" >
    <string>info</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>quit</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="ZlMd5Check.qrc" >:/icons/Icons/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ZlMd5Check.qrc" />
 </resources>
 <connections/>
</ui>
